src/utils/log.ts

```diff
@@ -20,6 +20,7 @@ export enum LogType {
   bidi = 'bidi',
   cdp = 'cdp',
   debug = 'debug',
+  debugError = 'debug:error',
```

Comment:
We should do this for other places as well as currectly you can't listen on all debug events + sub debug at the same time via `DEBUG=<bidi>:debug:*` (this does not include `<bidi>:debug`)